app/controllers/manual_document_attachments_controller.rb

```diff
@@ -41,21 +64,21 @@ def update
 
 private
 
-  def services
+  def repository
     if current_user_is_gds_editor?
-      gds_editor_services
+      gds_editor_repository
     else
-      organisational_services
+      organisational_repository
     end
   end
 
-  def gds_editor_services
-    ManualDocumentAttachmentServiceRegistry.new
+  def gds_editor_repository
+    RepositoryRegistry.create.manual_repository
   end
 
-  def organisational_services
-    OrganisationalManualDocumentAttachmentServiceRegistry.new(
-      organisation_slug: current_organisation_slug,
-    )
+  def organisational_repository
+    manual_repository_factory = RepositoryRegistry.create
```

Comment:
There's probably some additional indirection we can remove here, but I think we'll see it if we refactor all of the controllers to use this pattern.